packages/plugins/plugin-navtree/src/components/NavTree/NavTree.stories.tsx

```diff
@@ -36,13 +38,54 @@ faker.seed(1234);
 
 const StoryState = defineCapability<{ tab: string }>('story-state');
 
+// TODO(burdon): How to adjust existing surfaces for this test?
+// TODO(burdon): How to toggle attention in this test?
+// TODO(burdon): Fix outline (e.g., button in sidebar nav is clipped when focused).
+// TODO(burdon): Consider similar containment of: Table, Sheet, Kanban Column, Form, etc.
+
+const container = 'flex flex-col grow gap-2 p-4 rounded-md';
+
+const TestPanel = () => {
+  // TODO(burdon): Where should attention be applied?
+  const attentionAttrs = useAttendableAttributes('test');
+
+  return (
+    <StackItem.Content toolbar classNames='w-full'>
+      {/* TODO(burdon): Should the separator be applied by StackItem.Content? */}
```

Comment:
Bear in mind `StackItem` is upstream of, i.e. used by:
- Planks
- Stack sections
- Kanban columns
- Kanban cards

StackItem can apply styles relevant only to some of these, but I would assert we should apply typical separation of concerns, that it’s reasonable for Kanban, StackPlugin, and Deck to give concern-specific styles to their `StackItem`s and `StackItem.Content`s.

Reply:
i'm just pulling this in as a devDependency for this story.